utils/vpr_io_place.py

```diff
@@ -22,6 +22,7 @@ def __init__(self):
         self.inputs = set()
         self.outputs = set()
         self.net_to_block = None
+        self.block_to_inst = {}
```

Comment:
Just saw this, probably it is redundant as it is already defined below.

Reply:
Right... fixed.